### PR TITLE
the Rock Exchange

### PR DESCRIPTION
**Pull request by petko-happening**

### CODE REVIEW

File: cryptotik/binance.py [hunk 1]
```diff
@@ -39,17 +39,6 @@ def api(cls, url, params):
         except requests.exceptions.RequestException as e:
             raise APIError(e)
 
-    def get_nonce(self):
```
Comment (peerchemist) [inline cryptotik/binance.py:42]: Why is there a binance change inserted here?

Comment (petko-happening) [reply cryptotik/binance.py:42]: Realized the method was not needed, and was inserted by mistake. Didn't bother to commit it separately. Sorry!

Comment (peerchemist) [reply cryptotik/binance.py:42]: Binance API does not need nonce?

Comment (petko-happening) [reply cryptotik/binance.py:42]: no, it uses timestamp instead.

Comment (peerchemist) [reply cryptotik/binance.py:42]: Okay then.

**Pull request merged**